src/components/data_comps/desp/desp_comp_mod.F90

```diff
@@ -427,6 +427,13 @@ subroutine desp_comp_run(EClock, case_name, pause_sig, atm_resume,          &
 
     call t_startf('desp_mode')
 
+    if (.not. ANY(pause_sig)) then
+      if ((my_task == master_task) .and. (loglevel > 0)) then
```

Comment:
Can we get rid of the loglevel requirement in this and other locations in desp and just allways write these messages to the desp.log?